synapse/lib/storm.py

```diff
@@ -1838,7 +1864,7 @@ class LimitCmd(Cmd):
 
     def getArgParser(self):
         pars = Cmd.getArgParser(self)
-        pars.add_argument('count', type=int, help='The maximum number of nodes to yield.')
+        pars.add_argument('count', type='int', help='The maximum number of nodes to yield.')
```

Comment:
this is a breaking change we need to be aware of

Reply:
should only apply to python storm commands ( since pure storm commands had to be serializeable ) so i'd call it an internal change, but yea.  def need to be aware.